[I 2020-01-24 17:06:21.496 SingleUserNotebookApp extension:158] JupyterLab extension loaded from /opt/jupyterhub/lib64/python3.6/site-packages/jupyterlab
[I 2020-01-24 17:06:21.501 SingleUserNotebookApp extension:159] JupyterLab application directory is /opt/jupyterhub/share/jupyter/lab
[I 2020-01-24 17:06:21.895 SingleUserNotebookApp __init__:25] Loading lmod extension
[I 2020-01-24 17:06:21.899 SingleUserNotebookApp singleuser:561] Starting jupyterhub-singleuser server version 1.0.0
[I 2020-01-24 17:06:21.903 SingleUserNotebookApp log:174] 302 GET /user/guest160/ -> /user/guest160/tree? (@132.219.137.246) 1.55ms
[I 2020-01-24 17:06:21.909 SingleUserNotebookApp notebookapp:1875] Serving notebooks from local directory: /home/guest160
[I 2020-01-24 17:06:21.910 SingleUserNotebookApp notebookapp:1875] The Jupyter Notebook is running at:
[I 2020-01-24 17:06:21.910 SingleUserNotebookApp notebookapp:1875] http://helk80n02.helios.calculquebec.ca:58505/user/guest160/
[I 2020-01-24 17:06:21.910 SingleUserNotebookApp notebookapp:1876] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[I 2020-01-24 17:06:21.923 SingleUserNotebookApp singleuser:542] Updating Hub with activity every 300 seconds
[I 2020-01-24 17:06:22.155 SingleUserNotebookApp log:174] 302 GET /user/guest160/lab? -> /hub/api/oauth2/authorize?client_id=jupyterhub-user-guest160&redirect_uri=%2Fuser%2Fguest160%2Foauth_callback&response_type=code&state=[secret] (@69.28.236.4) 2.06ms
[I 2020-01-24 17:06:22.303 SingleUserNotebookApp auth:981] Logged-in user {'kind': 'user', 'name': 'guest160', 'admin': False, 'groups': [], 'server': '/user/guest160/', 'pending': None, 'created': '2020-01-08T21:36:15.420535Z', 'last_activity': '2020-01-24T17:06:22.297139Z', 'servers': None}
[I 2020-01-24 17:06:22.303 SingleUserNotebookApp log:174] 302 GET /user/guest160/oauth_callback?code=[secret]&state=[secret] -> /user/guest160/lab? (@69.28.236.4) 47.84ms
[I 2020-01-24 17:06:22.345 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab? (guest160@69.28.236.4) 6.17ms
[I 2020-01-24 17:06:26.704 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/kernelspecs?1579885587056 (guest160@69.28.236.4) 33.81ms
[I 2020-01-24 17:06:26.705 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885587054 (guest160@69.28.236.4) 5.25ms
[I 2020-01-24 17:06:26.764 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/?1579885587064 (guest160@69.28.236.4) 29.72ms
[I 2020-01-24 17:06:26.766 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885587057 (guest160@69.28.236.4) 0.98ms
[I 2020-01-24 17:06:26.773 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/kernelspecs?1579885587160 (guest160@69.28.236.4) 1.98ms
[I 2020-01-24 17:06:26.774 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885587159 (guest160@69.28.236.4) 1.09ms
[I 2020-01-24 17:06:26.826 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885587209 (guest160@69.28.236.4) 0.96ms
[I 2020-01-24 17:06:26.989 SingleUserNotebookApp log:174] 200 GET /user/guest160/metrics?1579885587376 (guest160@69.28.236.4) 7.35ms
[I 2020-01-24 17:06:28.036 SingleUserNotebookApp log:174] 200 GET /user/guest160/lmod/avail (guest160@69.28.236.4) 1013.25ms
[I 2020-01-24 17:06:28.042 SingleUserNotebookApp build_handler:49] Build is up to date
[I 2020-01-24 17:06:28.255 SingleUserNotebookApp log:174] 200 GET /user/guest160/lmod/list (guest160@69.28.236.4) 217.19ms
[I 2020-01-24 17:06:28.359 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/nbconvert?1579885587427 (guest160@69.28.236.4) 319.48ms
[I 2020-01-24 17:06:28.410 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/nbconvert?1579885587447 (guest160@69.28.236.4) 369.16ms
[I 2020-01-24 17:06:28.641 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885587485 (guest160@69.28.236.4) 599.21ms
[I 2020-01-24 17:06:28.641 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/build?1579885587407 (guest160@69.28.236.4) 1629.37ms
[I 2020-01-24 17:06:28.651 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587595 (guest160@69.28.236.4) 7.16ms
[I 2020-01-24 17:06:28.653 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587597 (guest160@69.28.236.4) 7.76ms
[I 2020-01-24 17:06:28.654 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587598 (guest160@69.28.236.4) 8.42ms
[I 2020-01-24 17:06:28.693 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587599 (guest160@69.28.236.4) 1.97ms
[I 2020-01-24 17:06:28.695 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587600 (guest160@69.28.236.4) 1.99ms
[I 2020-01-24 17:06:28.701 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587601 (guest160@69.28.236.4) 1.93ms
[I 2020-01-24 17:06:28.705 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587604 (guest160@69.28.236.4) 2.95ms
[I 2020-01-24 17:06:28.706 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587603 (guest160@69.28.236.4) 3.59ms
[I 2020-01-24 17:06:28.734 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587606 (guest160@69.28.236.4) 1.94ms
[I 2020-01-24 17:06:28.737 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587605 (guest160@69.28.236.4) 1.97ms
[I 2020-01-24 17:06:28.743 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587608 (guest160@69.28.236.4) 1.92ms
[I 2020-01-24 17:06:28.747 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587607 (guest160@69.28.236.4) 2.83ms
[I 2020-01-24 17:06:28.748 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587610 (guest160@69.28.236.4) 3.46ms
[I 2020-01-24 17:06:28.775 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587610 (guest160@69.28.236.4) 1.95ms
[I 2020-01-24 17:06:28.781 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587611 (guest160@69.28.236.4) 1.91ms
[I 2020-01-24 17:06:28.783 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587612 (guest160@69.28.236.4) 1.96ms
[I 2020-01-24 17:06:28.787 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587613 (guest160@69.28.236.4) 2.93ms
[I 2020-01-24 17:06:28.788 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587615 (guest160@69.28.236.4) 3.56ms
[I 2020-01-24 17:06:28.811 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587615 (guest160@69.28.236.4) 1.95ms
[I 2020-01-24 17:06:28.824 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587617 (guest160@69.28.236.4) 1.96ms
[I 2020-01-24 17:06:28.833 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587626 (guest160@69.28.236.4) 2.84ms
[I 2020-01-24 17:06:28.834 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/settings/@jupyterlab/docmanager-extension:plugin?1579885587618 (guest160@69.28.236.4) 3.65ms
[I 2020-01-24 17:06:28.836 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885587630 (guest160@69.28.236.4) 1.28ms
[I 2020-01-24 17:06:28.857 SingleUserNotebookApp log:174] 200 GET /user/guest160/lab/api/workspaces/lab?1579885587641 (guest160@69.28.236.4) 13.02ms
[I 2020-01-24 17:06:28.869 SingleUserNotebookApp log:174] 200 GET /user/guest160/metrics?1579885588186 (guest160@69.28.236.4) 9.83ms
[I 2020-01-24 17:06:28.928 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?1579885589313 (guest160@69.28.236.4) 5.60ms
[I 2020-01-24 17:06:28.929 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb?content=0&1579885589315 (guest160@69.28.236.4) 5.63ms
[I 2020-01-24 17:06:28.933 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tfenv.sh?content=0&1579885589317 (guest160@69.28.236.4) 1.93ms
[I 2020-01-24 17:06:28.974 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885589364 (guest160@69.28.236.4) 4.90ms
[I 2020-01-24 17:06:29.123 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885589504 (guest160@69.28.236.4) 1.03ms
[I 2020-01-24 17:06:29.501 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885589894 (guest160@69.28.236.4) 0.98ms
[I 2020-01-24 17:06:29.592 SingleUserNotebookApp management:320] New terminal with automatic name: 1
[I 2020-01-24 17:06:29.593 SingleUserNotebookApp log:174] 200 POST /user/guest160/api/terminals?1579885589939 (guest160@69.28.236.4) 44.93ms
[I 2020-01-24 17:06:29.626 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885590006 (guest160@69.28.236.4) 6.94ms
[I 2020-01-24 17:06:29.701 SingleUserNotebookApp log:174] 101 GET /user/guest160/terminals/websocket/1?token=[secret] (guest160@69.28.236.4) 1.33ms
TermSocket.open: 1
TermSocket.open: Opened 1
[I 2020-01-24 17:06:30.391 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb?type=notebook&content=1&1579885590119 (guest160@69.28.236.4) 666.55ms
[I 2020-01-24 17:06:30.406 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tfenv.sh?format=text&type=file&content=1&1579885590120 (guest160@69.28.236.4) 14.23ms
[I 2020-01-24 17:06:30.411 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885590142 (guest160@69.28.236.4) 17.64ms
[I 2020-01-24 17:06:30.414 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885590645 (guest160@69.28.236.4) 20.42ms
[I 2020-01-24 17:06:30.558 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb/checkpoints?1579885590903 (guest160@69.28.236.4) 49.09ms
[I 2020-01-24 17:06:30.560 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb/checkpoints?1579885590904 (guest160@69.28.236.4) 1.28ms
[I 2020-01-24 17:06:30.573 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb/checkpoints?1579885590966 (guest160@69.28.236.4) 1.25ms
[I 2020-01-24 17:06:30.667 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tfenv.sh/checkpoints?1579885591011 (guest160@69.28.236.4) 47.34ms
[I 2020-01-24 17:06:30.667 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tfenv.sh/checkpoints?1579885591009 (guest160@69.28.236.4) 47.28ms
[I 2020-01-24 17:06:30.670 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tfenv.sh/checkpoints?1579885591010 (guest160@69.28.236.4) 2.18ms
[I 2020-01-24 17:06:30.707 SingleUserNotebookApp kernelmanager:172] Kernel started: f8630d64-fea4-4365-a618-0942006398f6
[I 2020-01-24 17:06:30.709 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/sessions?1579885591029 (guest160@69.28.236.4) 39.77ms
[I 2020-01-24 17:06:32.174 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/f8630d64-fea4-4365-a618-0942006398f6/channels?session_id=a6ff18b1-161b-4041-a9e9-bb8d66df2ede&token=[secret] (guest160@69.28.236.4) 1358.45ms
[I 2020-01-24 17:06:32.277 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/f8630d64-fea4-4365-a618-0942006398f6/channels?session_id=ea4f24ab-7c93-411b-8207-7f7cf9120b6a&token=[secret] (guest160@69.28.236.4) 2.66ms
[I 2020-01-24 17:06:32.403 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885592794 (guest160@69.28.236.4) 1.31ms
[I 2020-01-24 17:06:33.908 SingleUserNotebookApp log:174] 200 GET /user/guest160/metrics?1579885594280 (guest160@69.28.236.4) 12.28ms
[I 2020-01-24 17:06:36.876 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885597262 (guest160@69.28.236.4) 1.03ms
[I 2020-01-24 17:06:40.628 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885601014 (guest160@69.28.236.4) 6.08ms
[I 2020-01-24 17:06:42.451 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885602842 (guest160@69.28.236.4) 1.59ms
[I 2020-01-24 17:06:44.613 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885604988 (guest160@69.28.236.4) 4.51ms
[I 2020-01-24 17:06:45.674 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885606055 (guest160@69.28.236.4) 4.58ms
[I 2020-01-24 17:06:46.415 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885606801 (guest160@69.28.236.4) 4.46ms
[I 2020-01-24 17:06:46.926 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885607313 (guest160@69.28.236.4) 1.00ms
[I 2020-01-24 17:06:47.768 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885608146 (guest160@69.28.236.4) 4.58ms
[I 2020-01-24 17:06:48.271 SingleUserNotebookApp log:174] 200 GET /user/guest160/metrics?1579885608642 (guest160@69.28.236.4) 11.53ms
[I 2020-01-24 17:06:50.680 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885611069 (guest160@69.28.236.4) 4.94ms
[I 2020-01-24 17:06:52.523 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885612888 (guest160@69.28.236.4) 1.21ms
[I 2020-01-24 17:06:55.977 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885616358 (guest160@69.28.236.4) 5.17ms
[I 2020-01-24 17:06:56.984 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885617372 (guest160@69.28.236.4) 1.28ms
[I 2020-01-24 17:06:57.394 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885617785 (guest160@69.28.236.4) 3.81ms
[I 2020-01-24 17:06:58.095 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885618476 (guest160@69.28.236.4) 4.12ms
[I 2020-01-24 17:06:59.058 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885619440 (guest160@69.28.236.4) 4.06ms
[I 2020-01-24 17:06:59.581 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885619963 (guest160@69.28.236.4) 4.08ms
[I 2020-01-24 17:07:00.735 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885621116 (guest160@69.28.236.4) 6.01ms
[I 2020-01-24 17:07:02.596 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885622959 (guest160@69.28.236.4) 1.53ms
[I 2020-01-24 17:07:05.112 SingleUserNotebookApp multikernelmanager:127] Kernel shutdown: f8630d64-fea4-4365-a618-0942006398f6
[I 2020-01-24 17:07:05.114 SingleUserNotebookApp log:174] 204 DELETE /user/guest160/api/sessions/62a9b2f8-9fd2-458c-bc9b-b40d388e1f21?1579885624892 (guest160@69.28.236.4) 608.64ms
[I 2020-01-24 17:07:05.527 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885625919 (guest160@69.28.236.4) 1.60ms
[I 2020-01-24 17:07:06.132 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885626509 (guest160@69.28.236.4) 5.12ms
[I 2020-01-24 17:07:06.861 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885627247 (guest160@69.28.236.4) 4.09ms
[I 2020-01-24 17:07:07.034 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?1579885627423 (guest160@69.28.236.4) 1.27ms
[I 2020-01-24 17:07:07.926 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885628306 (guest160@69.28.236.4) 4.39ms
[I 2020-01-24 17:07:08.433 SingleUserNotebookApp management:185] EOF on FD 11; stopping reading
[I 2020-01-24 17:07:08.534 SingleUserNotebookApp management:338] Terminal 1 closed
Websocket closed
[I 2020-01-24 17:07:09.227 SingleUserNotebookApp log:174] 204 PUT /user/guest160/lab/api/workspaces/lab?1579885629614 (guest160@69.28.236.4) 4.19ms
[I 2020-01-24 17:07:10.792 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/?content=1&1579885631173 (guest160@69.28.236.4) 5.95ms
[I 2020-01-24 17:07:15.666 SingleUserNotebookApp log:174] 302 GET /user/guest160/logout -> /hub/logout (guest160@69.28.236.4) 1.21ms
[I 2020-01-24 17:07:15.679 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?1579885636073 (guest160@69.28.236.4) 1.39ms
[I 2020-01-24 17:07:23.380 SingleUserNotebookApp log:174] 302 GET /user/guest160/ -> /user/guest160/tree? (@69.28.236.4) 0.76ms
[I 2020-01-24 17:07:23.529 SingleUserNotebookApp log:174] 200 GET /user/guest160/tree? (guest160@69.28.236.4) 95.32ms
[I 2020-01-24 17:07:25.447 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/tree?_=1579885645722 (guest160@69.28.236.4) 4.42ms
[I 2020-01-24 17:07:25.455 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/kernelspecs (guest160@69.28.236.4) 6.20ms
[I 2020-01-24 17:07:25.455 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645724 (guest160@69.28.236.4) 5.96ms
[I 2020-01-24 17:07:25.458 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579885645723 (guest160@69.28.236.4) 7.92ms
[I 2020-01-24 17:07:25.459 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645725 (guest160@69.28.236.4) 7.76ms
[I 2020-01-24 17:07:25.461 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645726 (guest160@69.28.236.4) 1.10ms
[I 2020-01-24 17:07:25.515 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645727 (guest160@69.28.236.4) 5.91ms
[I 2020-01-24 17:07:26.248 SingleUserNotebookApp log:174] 200 GET /user/guest160/lmod/savelist (guest160@69.28.236.4) 241.65ms
[I 2020-01-24 17:07:37.639 SingleUserNotebookApp log:174] 200 GET /user/guest160/notebooks/Untitled.ipynb (guest160@69.28.236.4) 208.81ms
[I 2020-01-24 17:07:39.946 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/notebook?_=1579885660236 (guest160@69.28.236.4) 3.52ms
[I 2020-01-24 17:07:39.951 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579885660237 (guest160@69.28.236.4) 3.38ms
[I 2020-01-24 17:07:40.272 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb?type=notebook&_=1579885660238 (guest160@69.28.236.4) 268.53ms
[W 2020-01-24 17:07:40.289 SingleUserNotebookApp log:174] 404 GET /user/guest160/nbextensions/widgets/notebook/js/extension.js?v=20200124170621 (guest160@69.28.236.4) 13.03ms
[I 2020-01-24 17:07:40.425 SingleUserNotebookApp kernelmanager:172] Kernel started: 9051d21d-ba38-49ec-9b87-914c6931a922
[I 2020-01-24 17:07:40.426 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/sessions (guest160@69.28.236.4) 24.23ms
[I 2020-01-24 17:07:40.429 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb/checkpoints?_=1579885660239 (guest160@69.28.236.4) 1.95ms
[I 2020-01-24 17:07:41.506 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/9051d21d-ba38-49ec-9b87-914c6931a922/channels?session_id=2638b1ca02b742dd8c75c453563496da (guest160@69.28.236.4) 968.03ms
[I 2020-01-24 17:07:45.964 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645728 (guest160@69.28.236.4) 1.53ms
[I 2020-01-24 17:07:45.966 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645729 (guest160@69.28.236.4) 1.19ms
[I 2020-01-24 17:07:46.014 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645730 (guest160@69.28.236.4) 5.97ms
[I 2020-01-24 17:07:48.354 SingleUserNotebookApp kernelmanager:217] Starting buffering for 9051d21d-ba38-49ec-9b87-914c6931a922:2638b1ca02b742dd8c75c453563496da
[I 2020-01-24 17:07:51.127 SingleUserNotebookApp multikernelmanager:127] Kernel shutdown: 9051d21d-ba38-49ec-9b87-914c6931a922
[I 2020-01-24 17:07:51.128 SingleUserNotebookApp log:174] 204 DELETE /user/guest160/api/sessions/46975676-dcce-490c-b89d-8be390f14141 (guest160@69.28.236.4) 505.50ms
[I 2020-01-24 17:07:51.186 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645731 (guest160@69.28.236.4) 1.54ms
[I 2020-01-24 17:07:51.244 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645732 (guest160@69.28.236.4) 5.96ms
[I 2020-01-24 17:07:55.811 SingleUserNotebookApp log:174] 200 PATCH /user/guest160/api/contents/Untitled.ipynb (guest160@69.28.236.4) 6.78ms
[I 2020-01-24 17:07:55.865 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645733 (guest160@69.28.236.4) 7.29ms
[I 2020-01-24 17:08:02.334 SingleUserNotebookApp handlers:153] Creating new notebook in 
[I 2020-01-24 17:08:02.854 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/contents (guest160@69.28.236.4) 520.94ms
[I 2020-01-24 17:08:03.019 SingleUserNotebookApp log:174] 200 GET /user/guest160/notebooks/Untitled.ipynb?kernel_name=python3 (guest160@69.28.236.4) 123.57ms
[I 2020-01-24 17:08:03.469 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645734 (guest160@69.28.236.4) 1.25ms
[I 2020-01-24 17:08:03.471 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645735 (guest160@69.28.236.4) 1.13ms
[I 2020-01-24 17:08:03.532 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645736 (guest160@69.28.236.4) 7.50ms
[I 2020-01-24 17:08:04.442 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/notebook?_=1579885684018 (guest160@69.28.236.4) 4.69ms
[I 2020-01-24 17:08:04.445 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579885684019 (guest160@69.28.236.4) 6.26ms
[I 2020-01-24 17:08:04.776 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb?type=notebook&_=1579885684020 (guest160@69.28.236.4) 293.49ms
[W 2020-01-24 17:08:04.785 SingleUserNotebookApp log:174] 404 GET /user/guest160/nbextensions/widgets/notebook/js/extension.js?v=20200124170621 (guest160@69.28.236.4) 4.63ms
[I 2020-01-24 17:08:04.892 SingleUserNotebookApp kernelmanager:172] Kernel started: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 17:08:04.894 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/sessions (guest160@69.28.236.4) 23.97ms
[I 2020-01-24 17:08:04.897 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Untitled.ipynb/checkpoints?_=1579885684021 (guest160@69.28.236.4) 1.87ms
[I 2020-01-24 17:08:05.950 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/channels?session_id=83be9d50825d4852825e3d5bcdfb5fc0 (guest160@69.28.236.4) 944.45ms
[I 2020-01-24 17:08:13.829 SingleUserNotebookApp log:174] 200 PATCH /user/guest160/api/contents/Untitled.ipynb (guest160@69.28.236.4) 6.78ms
[I 2020-01-24 17:08:13.883 SingleUserNotebookApp log:174] 200 PATCH /user/guest160/api/sessions/cc8f4bb1-6bdd-4a73-a67c-15a0381d3b97 (guest160@69.28.236.4) 2.09ms
[I 2020-01-24 17:10:04.946 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579885684022 (guest160@69.28.236.4) 1.82ms
[I 2020-01-24 17:10:05.013 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 17:10:05.306 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 294.58ms
[I 2020-01-24 17:26:15.485 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645737 (guest160@69.28.236.4) 1.21ms
[I 2020-01-24 17:26:15.520 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645738 (guest160@69.28.236.4) 0.92ms
[I 2020-01-24 17:26:15.561 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645739 (guest160@69.28.236.4) 6.32ms
[I 2020-01-24 17:27:21.560 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645740 (guest160@69.28.236.4) 2.34ms
[I 2020-01-24 17:27:21.560 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645741 (guest160@69.28.236.4) 2.05ms
[I 2020-01-24 17:27:21.612 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645742 (guest160@69.28.236.4) 5.42ms
[I 2020-01-24 17:27:31.873 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645743 (guest160@69.28.236.4) 1.17ms
[I 2020-01-24 17:27:31.874 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645744 (guest160@69.28.236.4) 0.88ms
[I 2020-01-24 17:27:31.920 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645745 (guest160@69.28.236.4) 4.92ms
[I 2020-01-24 17:28:05.172 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645746 (guest160@69.28.236.4) 1.65ms
[I 2020-01-24 17:28:05.174 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645747 (guest160@69.28.236.4) 0.97ms
[I 2020-01-24 17:28:05.225 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645748 (guest160@69.28.236.4) 5.66ms
[I 2020-01-24 17:28:05.311 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579885684023 (guest160@69.28.236.4) 1.40ms
[I 2020-01-24 17:28:05.350 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 17:28:05.668 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 318.84ms
[I 2020-01-24 17:29:32.306 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 17:29:32.307 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.52ms
[I 2020-01-24 17:29:33.946 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645749 (guest160@69.28.236.4) 1.36ms
[I 2020-01-24 17:29:33.981 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645750 (guest160@69.28.236.4) 0.90ms
[I 2020-01-24 17:29:34.007 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645751 (guest160@69.28.236.4) 5.46ms
[I 2020-01-24 17:29:45.295 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645752 (guest160@69.28.236.4) 1.55ms
[I 2020-01-24 17:29:45.298 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645753 (guest160@69.28.236.4) 1.10ms
[I 2020-01-24 17:29:45.342 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645754 (guest160@69.28.236.4) 6.50ms
[I 2020-01-24 17:29:48.322 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/my_tf_env?type=directory&_=1579885645755 (guest160@69.28.236.4) 8.71ms
[I 2020-01-24 17:29:50.524 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645756 (guest160@69.28.236.4) 5.05ms
[I 2020-01-24 17:30:04.888 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579885684024 (guest160@69.28.236.4) 1.36ms
[I 2020-01-24 17:30:04.931 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 17:30:05.482 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 552.22ms
[I 2020-01-24 17:30:47.240 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 17:30:47.241 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.31ms
[I 2020-01-24 17:32:04.981 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579885684025 (guest160@69.28.236.4) 30.95ms
[I 2020-01-24 17:32:05.042 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 17:32:05.531 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 490.06ms
[I 2020-01-24 17:32:51.767 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645757 (guest160@69.28.236.4) 1.33ms
[I 2020-01-24 17:32:51.802 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645758 (guest160@69.28.236.4) 0.92ms
[I 2020-01-24 17:32:51.833 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645759 (guest160@69.28.236.4) 6.65ms
[I 2020-01-24 17:32:59.419 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 17:32:59.420 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.30ms
[I 2020-01-24 17:33:11.263 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645760 (guest160@69.28.236.4) 1.37ms
[I 2020-01-24 17:33:11.264 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645761 (guest160@69.28.236.4) 0.89ms
[I 2020-01-24 17:33:11.308 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645762 (guest160@69.28.236.4) 5.47ms
[I 2020-01-24 17:33:22.796 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645763 (guest160@69.28.236.4) 1.07ms
[I 2020-01-24 17:33:22.800 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645764 (guest160@69.28.236.4) 0.85ms
[I 2020-01-24 17:33:22.842 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645765 (guest160@69.28.236.4) 5.06ms
[I 2020-01-24 17:33:24.615 SingleUserNotebookApp log:174] 200 GET /user/guest160/notebooks/tf%20install.ipynb (guest160@69.28.236.4) 123.02ms
[I 2020-01-24 17:33:25.315 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/notebook?_=1579887205588 (guest160@69.28.236.4) 2.86ms
[I 2020-01-24 17:33:25.336 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579887205589 (guest160@69.28.236.4) 2.64ms
[I 2020-01-24 17:33:25.669 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tf%20install.ipynb?type=notebook&_=1579887205590 (guest160@69.28.236.4) 286.34ms
[W 2020-01-24 17:33:25.677 SingleUserNotebookApp log:174] 404 GET /user/guest160/nbextensions/widgets/notebook/js/extension.js?v=20200124170621 (guest160@69.28.236.4) 5.03ms
[I 2020-01-24 17:33:25.846 SingleUserNotebookApp kernelmanager:172] Kernel started: 38e7ef98-e030-4962-82c5-b4e6de5f8dcd
[I 2020-01-24 17:33:25.848 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/sessions (guest160@69.28.236.4) 41.12ms
[I 2020-01-24 17:33:25.851 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tf%20install.ipynb/checkpoints?_=1579887205591 (guest160@69.28.236.4) 2.37ms
[I 2020-01-24 17:33:26.915 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/38e7ef98-e030-4962-82c5-b4e6de5f8dcd/channels?session_id=082b616a10b0496198f851b674de5321 (guest160@69.28.236.4) 956.71ms
[I 2020-01-24 17:34:04.052 SingleUserNotebookApp kernelmanager:217] Starting buffering for 38e7ef98-e030-4962-82c5-b4e6de5f8dcd:082b616a10b0496198f851b674de5321
[I 2020-01-24 17:34:04.069 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/tf%20install.ipynb?content=0&_=1579887205592 (guest160@69.28.236.4) 1.65ms
[I 2020-01-24 17:34:04.347 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645766 (guest160@69.28.236.4) 1.31ms
[I 2020-01-24 17:34:04.349 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645767 (guest160@69.28.236.4) 0.92ms
[I 2020-01-24 17:34:04.393 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645768 (guest160@69.28.236.4) 5.49ms
[I 2020-01-24 17:34:05.306 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579885684026 (guest160@69.28.236.4) 1.33ms
[I 2020-01-24 17:34:05.350 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 17:34:05.665 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 315.77ms
[I 2020-01-24 17:34:07.222 SingleUserNotebookApp multikernelmanager:127] Kernel shutdown: 38e7ef98-e030-4962-82c5-b4e6de5f8dcd
[I 2020-01-24 17:34:07.223 SingleUserNotebookApp log:174] 204 DELETE /user/guest160/api/sessions/be4ed214-7d5a-4314-9878-9640c141696e (guest160@69.28.236.4) 506.11ms
[I 2020-01-24 17:34:07.263 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645769 (guest160@69.28.236.4) 1.43ms
[I 2020-01-24 17:34:07.313 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645770 (guest160@69.28.236.4) 6.52ms
[I 2020-01-24 17:34:32.450 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579885645771 (guest160@69.28.236.4) 3.08ms
[I 2020-01-24 17:34:32.451 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579885645772 (guest160@69.28.236.4) 2.43ms
[I 2020-01-24 17:34:32.518 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579885645773 (guest160@69.28.236.4) 5.40ms
[I 2020-01-24 17:35:13.122 SingleUserNotebookApp kernelmanager:217] Starting buffering for 28d50fe0-bc97-4072-93ed-950545f48441:83be9d50825d4852825e3d5bcdfb5fc0
[I 2020-01-24 18:07:35.765 SingleUserNotebookApp log:174] 200 GET /user/guest160/tree (guest160@69.28.236.4) 40.79ms
[I 2020-01-24 18:07:37.304 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/tree?_=1579889257621 (guest160@69.28.236.4) 4.02ms
[I 2020-01-24 18:07:37.341 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579889257622 (guest160@69.28.236.4) 3.34ms
[I 2020-01-24 18:07:37.346 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257623 (guest160@69.28.236.4) 1.13ms
[I 2020-01-24 18:07:37.369 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257624 (guest160@69.28.236.4) 2.46ms
[I 2020-01-24 18:07:37.374 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257625 (guest160@69.28.236.4) 1.10ms
[I 2020-01-24 18:07:37.429 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257626 (guest160@69.28.236.4) 6.15ms
[I 2020-01-24 18:07:38.147 SingleUserNotebookApp log:174] 200 GET /user/guest160/lmod/savelist (guest160@69.28.236.4) 199.89ms
[I 2020-01-24 18:07:42.550 SingleUserNotebookApp log:174] 200 GET /user/guest160/notebooks/Data%20Explore.ipynb (guest160@69.28.236.4) 130.07ms
[I 2020-01-24 18:07:42.948 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/notebook?_=1579889263236 (guest160@69.28.236.4) 3.44ms
[I 2020-01-24 18:07:42.950 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579889263237 (guest160@69.28.236.4) 4.84ms
[I 2020-01-24 18:07:43.476 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?type=notebook&_=1579889263238 (guest160@69.28.236.4) 451.14ms
[W 2020-01-24 18:07:43.483 SingleUserNotebookApp log:174] 404 GET /user/guest160/nbextensions/widgets/notebook/js/extension.js?v=20200124170621 (guest160@69.28.236.4) 3.84ms
[I 2020-01-24 18:07:43.606 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/sessions (guest160@69.28.236.4) 1.39ms
[I 2020-01-24 18:07:43.609 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb/checkpoints?_=1579889263239 (guest160@69.28.236.4) 1.14ms
[I 2020-01-24 18:07:43.725 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/channels?session_id=c63d506cc04d456d9f417946233d1457 (guest160@69.28.236.4) 2.54ms
[W 2020-01-24 18:08:18.201 SingleUserNotebookApp handlers:320] IOPub data rate exceeded.
    The notebook server will temporarily stop sending output
    to the client in order to avoid crashing it.
    To change this limit, set the config variable
    `--NotebookApp.iopub_data_rate_limit`.
    
    Current values:
    NotebookApp.iopub_data_rate_limit=1000000.0 (bytes/sec)
    NotebookApp.rate_limit_window=3.0 (secs)
    
[I 2020-01-24 18:08:18.204 SingleUserNotebookApp restarter:110] KernelRestarter: restarting kernel (1/5), keep random ports
kernel 28d50fe0-bc97-4072-93ed-950545f48441 restarted
[I 2020-01-24 18:08:18.220 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:18.220 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 2.26ms
[I 2020-01-24 18:08:21.218 SingleUserNotebookApp restarter:110] KernelRestarter: restarting kernel (2/5), keep random ports
kernel 28d50fe0-bc97-4072-93ed-950545f48441 restarted
[I 2020-01-24 18:08:26.208 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:26.209 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 2.72ms
[I 2020-01-24 18:08:26.329 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:26.330 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.50ms
[I 2020-01-24 18:08:26.471 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:26.472 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.49ms
[I 2020-01-24 18:08:26.642 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:26.643 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.44ms
[I 2020-01-24 18:08:26.804 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:26.804 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.40ms
[I 2020-01-24 18:08:30.430 SingleUserNotebookApp multikernelmanager:173] Kernel interrupted: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:30.431 SingleUserNotebookApp log:174] 204 POST /user/guest160/api/kernels/28d50fe0-bc97-4072-93ed-950545f48441/interrupt (guest160@69.28.236.4) 1.69ms
[I 2020-01-24 18:08:30.767 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257628 (guest160@69.28.236.4) 1.25ms
[I 2020-01-24 18:08:30.770 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257627 (guest160@69.28.236.4) 1.69ms
[I 2020-01-24 18:08:31.011 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257629 (guest160@69.28.236.4) 6.78ms
[I 2020-01-24 18:08:36.792 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579889263240 (guest160@69.28.236.4) 2.10ms
[I 2020-01-24 18:08:37.018 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 18:08:37.429 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 412.09ms
[I 2020-01-24 18:08:37.433 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579889263241 (guest160@69.28.236.4) 2.62ms
[I 2020-01-24 18:08:37.434 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579889263242 (guest160@69.28.236.4) 2.71ms
[I 2020-01-24 18:08:37.665 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/contents/Data%20Explore.ipynb/checkpoints (guest160@69.28.236.4) 9.37ms
[I 2020-01-24 18:08:37.669 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 18:08:38.617 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 950.80ms
[I 2020-01-24 18:08:38.618 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 18:08:39.080 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 1412.12ms
[I 2020-01-24 18:08:39.723 SingleUserNotebookApp kernelmanager:217] Starting buffering for 28d50fe0-bc97-4072-93ed-950545f48441:c63d506cc04d456d9f417946233d1457
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
Task exception was never retrieved
future: <Task finished coro=<WebSocketProtocol13.write_message.<locals>.wrapper() done, defined at /opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py:1102> exception=WebSocketClosedError()>
Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1104, in wrapper
    await fut
tornado.iostream.StreamClosedError: Stream is closed

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/jupyterhub/lib64/python3.6/site-packages/tornado/websocket.py", line 1106, in wrapper
    raise WebSocketClosedError()
tornado.websocket.WebSocketClosedError
[I 2020-01-24 18:08:42.749 SingleUserNotebookApp multikernelmanager:127] Kernel shutdown: 28d50fe0-bc97-4072-93ed-950545f48441
[I 2020-01-24 18:08:42.750 SingleUserNotebookApp log:174] 204 DELETE /user/guest160/api/sessions/cc8f4bb1-6bdd-4a73-a67c-15a0381d3b97 (guest160@69.28.236.4) 405.61ms
[I 2020-01-24 18:08:42.792 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257630 (guest160@69.28.236.4) 1.51ms
[I 2020-01-24 18:08:42.845 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257631 (guest160@69.28.236.4) 7.64ms
[I 2020-01-24 18:08:44.829 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/notebook?_=1579889325087 (guest160@69.28.236.4) 3.47ms
[I 2020-01-24 18:08:44.834 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/config/common?_=1579889325088 (guest160@69.28.236.4) 3.36ms
[I 2020-01-24 18:08:45.130 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?type=notebook&_=1579889325089 (guest160@69.28.236.4) 262.34ms
[W 2020-01-24 18:08:45.139 SingleUserNotebookApp log:174] 404 GET /user/guest160/nbextensions/widgets/notebook/js/extension.js?v=20200124170621 (guest160@69.28.236.4) 6.24ms
[I 2020-01-24 18:08:45.280 SingleUserNotebookApp kernelmanager:172] Kernel started: 926dfed7-0216-4321-b6c5-37759d5438ea
[I 2020-01-24 18:08:45.281 SingleUserNotebookApp log:174] 201 POST /user/guest160/api/sessions (guest160@69.28.236.4) 24.52ms
[I 2020-01-24 18:08:45.284 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb/checkpoints?_=1579889325090 (guest160@69.28.236.4) 1.87ms
[I 2020-01-24 18:08:46.227 SingleUserNotebookApp log:174] 101 GET /user/guest160/api/kernels/926dfed7-0216-4321-b6c5-37759d5438ea/channels?session_id=c0eb7ec6c5cf4e90bebe344549fa5263 (guest160@69.28.236.4) 837.22ms
[I 2020-01-24 18:10:28.325 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257632 (guest160@69.28.236.4) 1.51ms
[I 2020-01-24 18:10:28.361 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257633 (guest160@69.28.236.4) 0.92ms
[I 2020-01-24 18:10:28.384 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257634 (guest160@69.28.236.4) 5.70ms
[I 2020-01-24 18:10:45.278 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579889325091 (guest160@69.28.236.4) 1.34ms
[I 2020-01-24 18:10:45.352 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 18:10:45.904 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 553.36ms
[I 2020-01-24 18:11:23.607 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257635 (guest160@69.28.236.4) 1.64ms
[I 2020-01-24 18:11:23.608 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257636 (guest160@69.28.236.4) 0.97ms
[I 2020-01-24 18:11:23.685 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257637 (guest160@69.28.236.4) 11.12ms
[I 2020-01-24 18:12:45.298 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579889325092 (guest160@69.28.236.4) 33.04ms
[I 2020-01-24 18:12:45.356 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 18:12:45.832 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 477.97ms
[I 2020-01-24 18:14:45.340 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents/Data%20Explore.ipynb?content=0&_=1579889325093 (guest160@69.28.236.4) 2.95ms
[I 2020-01-24 18:14:45.454 SingleUserNotebookApp handlers:164] Saving file at /Data Explore.ipynb
[I 2020-01-24 18:14:45.751 SingleUserNotebookApp log:174] 200 PUT /user/guest160/api/contents/Data%20Explore.ipynb (guest160@69.28.236.4) 298.07ms
[I 2020-01-24 18:17:51.547 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257639 (guest160@69.28.236.4) 31.23ms
[I 2020-01-24 18:17:51.548 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257638 (guest160@69.28.236.4) 2.06ms
[I 2020-01-24 18:17:51.603 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257640 (guest160@69.28.236.4) 14.48ms
[I 2020-01-24 18:18:19.572 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257641 (guest160@69.28.236.4) 1.14ms
[I 2020-01-24 18:18:19.574 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257642 (guest160@69.28.236.4) 0.94ms
[I 2020-01-24 18:18:19.620 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257643 (guest160@69.28.236.4) 5.16ms
[I 2020-01-24 18:21:56.325 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257644 (guest160@69.28.236.4) 1.34ms
[I 2020-01-24 18:21:56.326 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257645 (guest160@69.28.236.4) 0.94ms
[I 2020-01-24 18:21:56.377 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257646 (guest160@69.28.236.4) 5.55ms
[I 2020-01-24 18:25:43.821 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257647 (guest160@69.28.236.4) 32.55ms
[I 2020-01-24 18:25:43.847 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257648 (guest160@69.28.236.4) 1.20ms
[I 2020-01-24 18:25:43.881 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257649 (guest160@69.28.236.4) 6.94ms
[I 2020-01-24 18:26:14.321 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257650 (guest160@69.28.236.4) 1.43ms
[I 2020-01-24 18:26:14.323 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257651 (guest160@69.28.236.4) 1.15ms
[I 2020-01-24 18:26:14.369 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257652 (guest160@69.28.236.4) 5.72ms
[I 2020-01-24 18:27:16.322 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/sessions?_=1579889257653 (guest160@69.28.236.4) 1.28ms
[I 2020-01-24 18:27:16.323 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/terminals?_=1579889257654 (guest160@69.28.236.4) 0.93ms
[I 2020-01-24 18:27:16.372 SingleUserNotebookApp log:174] 200 GET /user/guest160/api/contents?type=directory&_=1579889257655 (guest160@69.28.236.4) 4.89ms
[I 2020-01-24 18:31:47.300 SingleUserNotebookApp kernelmanager:217] Starting buffering for 926dfed7-0216-4321-b6c5-37759d5438ea:c0eb7ec6c5cf4e90bebe344549fa5263
slurmstepd: error: *** JOB 1381 ON helk80n02 CANCELLED AT 2020-01-24T19:04:35 ***
slurmstepd: error: Detected 1 oom-kill event(s) in step 1381.batch cgroup. Some of your processes may have been killed by the cgroup out-of-memory handler.
